bridge/nctalk/nctalk.go

```diff
@@ -177,6 +187,25 @@ func (b *Btalk) handleFiles(mmsg *config.Message, message *ocs.TalkRoomMessageDa
 	return nil
 }
 
+func (b *Btalk) handleSendingFile(msg *config.Message, r *Broom) error {
+	for _, f := range msg.Extra["file"] {
+		fi := f.(config.FileInfo)
+		if fi.URL != "" {
```

Comment:
small nitpick, make this inverted so we decrease the indents
```
if fi.URL == "" {
   continue
}
```

Reply:
:+1: I like it